src/components/Question.jsx

```diff
@@ -119,25 +119,46 @@ export default class Question extends React.Component {
     this.setState({ showIcon: false });
   };
 
-  render() {
+  _getHeaderClassName = () => {
     const question = this.props.question;
-    const subQuestion = question[Constants.HAS_SUBQUESTION];
+    const collapsible = this.props.collapsible;
     const options = this.context.options;
-    const questionComponent = this.renderQuestion(question);
-    if (FormUtils.isHidden(question)) {
-      return null;
-    }
+    const subQuestion = question[Constants.HAS_SUBQUESTION];
+
+    let headerClassName = [];
+
+    headerClassName.push(
+      Question.getEmphasizedOnRelevantClass(question),
+      "question-header"
+    );
+
+    if (FormUtils.isWizardStep(question))
+      headerClassName.push("wizard-step bg-primary text-white");
+    if (FormUtils.isEmphasised(question))
+      headerClassName.push(Question.getEmphasizedClass(question));
+    if (FormUtils.isSection(question))
+      headerClassName.push("section-background");
+    if (collapsible) headerClassName.push("cursor-pointer");
     if (
-      !FormUtils.isRelevant(question) &&
-      (this.context.options.debugMode ||
+      FormUtils.isRelevant(question) &&
```

Comment:
Is this really correct or it should be `!FormUtils.isRelevant(question)` ?

Reply:
![image](https://user-images.githubusercontent.com/3962237/186477889-d28bc616-faed-4321-9035-da93765ba380.png)

Reply:
Yes you are right, my bad... Changed